### ISSUE TITLE
Change font

### ISSUE THREAD

**Issue by DonSudo** [adamerose/vscode-markdown-wysiwyg#10]
This plugin is awesome🥰, but I hope that I can choose the font in preview🤩

**Comment by adamerose** [comment]
Agreed! Just changed this and published 0.7.7, it should inherit the editor font now.


Here's Wingdings

<img width="435" alt="image" src="https://github.com/adamerose/vscode-markdown-wysiwyg/assets/10884874/26800490-0d25-41fd-baa7-b0532176c4a4">

**Comment by adamerose** [comment]
Initially closed this after changing it to inherit editor font, but then realized that doesn't look very good since it's always mono. Will probably need to add an extra extension setting instead

**Comment by RagibHasin** [comment]
I think following the built-in `markdown.styles` setting automatically would be sufficient.
